### DISCUSSION TITLE
有什么方法可以挂到docker上吗？

### DISCUSSION THREAD

**Discussion by ShuoHuangTesting** [dundunnp/hamibot-auto_xuexiqiangguo#700] [Ideas]
目前用着一台手机当肉鸡，但是还是会忘记。

可以像techxuexi那样挂载到docker上再用pushplus登录吗？

目前有一台群辉DS920+，想挂上去一直跑着。

**Comment by Huan-Yunbing** [comment]
好像cookie存活有问题，貌似超过12小时cookie会失活

**Reply by ShuoHuangTesting** [reply]
对所以techxuexi是每次失效了就发二维码来登录，而且用的是web端。这个项目也停了很久了，基本不能用了。所以不知道手机上有没有什么骚操作还可以用。妈的就忘记了一天就被全社区通报，差点全家名字都上大字报了。

**Comment by Huan-Yunbing** [comment]
autojs 自己写脚本吧
